torch/fx/_utils.py

```diff
@@ -20,8 +20,10 @@ def format_name():
     if "print_output" not in kwargs:
         kwargs["print_output"] = False
 
-    if "colored" in kwargs and not sys.stdout.isatty():
-        kwargs["colored"] = False
+    _stream_stdout = getattr(sys, 'stdout')
```

Comment:
Is there a reason to do this? It seems equivalent to accessing sys.stdout

Reply:
> Is there a reason to do this? It seems equivalent to accessing sys.stdout

I changed `sys.stdout`  to an object that does not imitate `_io.TextIOWrapper` in my project, so there is no method `isatty` in this case and will raise AttributeError.

> torch._dynamo.exc.InternalTorchDynamoError: AttributeError: 'F' object has no attribute 'isatty'

Reply:
Specifically, why say `getattr(sys, 'stdout')` when `sys.stdout` seems like it would work

Reply:
yeah lint complaining about it

Reply:
> yeah lint complaining about it

My bad, sorry for the inconvenience. Unnecessary code has been removed in the latest commit in this PR.